docs/syscall_descriptions.md

```diff
@@ -84,6 +84,9 @@ Note: _all_ generated files (`*.const`, `*.go`, `*.h`) are checked-in with the
 Note: `make extract` extracts constants for all architectures which requires
 installed cross-compilers. If you get errors about missing compilers/libraries,
 try `sudo make install_prerequisites` or install equivalent package for your distro.
+The command install_reqrequisites will success even some of the installation failed,
+sudo apt-get update && sudo apt-get upgrade might be required to make this more
```

Comment:
`make install_reqrequisites` already does `apt-get update`.
Why is `upgrade` necessary? Is it?
Please surround commands with backticks, check out surrounding text.

Reply:
It is necessary because I have experienced the problem with apt-get install failed because of out-of-dated dependency packages:

The following packages have unmet dependencies:
gcc-aarch64-linux-gnu : Depends: gcc-8-aarch64-linux-gnu (>= 8.2.0-8~)
gcc-arm-linux-gnueabi : Depends: gcc-8-arm-linux-gnueabi (>= 8.2.0-8~)
gcc-powerpc64le-linux-gnu : Depends: gcc-8-powerpc64le-linux-gnu (>= 8.2.0-8~)
E: Unable to correct problems, you have held broken packages.

In such case I think we should mention that upgrade might be required as well

Reply:
And I am actually not sure if adding upgrade directly into prerequisites will be a good idea, there might be cases that a customer don't want to have their package upgraded?

Reply:
Agree. Doing `upgrade` implicitly is probably a bad thing.